tasks/tasks/scale_indicator.py

```diff
@@ -0,0 +1,76 @@
+from utils import download_rawfile, put_rawfile
+import pandas as pd
+import numpy as np
+import os
+import logging
+
+def scale_indicator(context):
+    logging.info(f"Scaling indicator {context.get('uuid')}")
+    data_paths=context.get('dataset').get('data_paths')
+    min_max_mapping={}
+    for path in data_paths:
+        # current_df=pd.read_parquet(path)
+        filename=path.split('/')[-1]
+        try:        
+            logging.info(f'downloading for s3 {path}')
+            download_rawfile(path, f"processing/{filename}")
+        except FileNotFoundError as e:
+            return {"success":False,'message':"File not found"}
+        
+        current_df=pd.read_parquet(f"processing/{filename}")
+
+        features = current_df.feature.unique()
+        for f in features:
+            feat = current_df[current_df["feature"] == f].copy()
```

Comment:
copy is slow and potentially wastes a lot of memory. Can we not just work on `feat` and not modify it?